app/src/main/res/layout-v1/theme_entry_card_n.xml

```diff
@@ -7,7 +7,7 @@
     android:foreground="?android:selectableItemBackground"
     card_view:cardBackgroundColor="@color/main_screen_card_background"
     card_view:cardCornerRadius="0dp"
-    card_view:cardElevation="0dp">
+    card_view:cardElevation="2dp">
```

Comment:
It really depends per person, you can actually keep that, and expose a 0dp elevation in dimens.xml so that a theme can overlay it for your personal liking. But N cards aren't really cards, but more screen-wide layouts which remove the traditional elevation.